create-reporter/index.test.js

```diff
@@ -71,57 +74,65 @@ function createError (name, message) {
   return err
 }
 
+function getStream (logger) {
+  let streamSym = Object.getOwnPropertySymbols(logger).find(it => {
```

Comment:
There is no adequate way to get stream from logger, so I decided to introduce this dirty hack to save tests fro stream guts. Don't think that this is sane solution.